enhancements/cluster-logging/container-log-throttling.md

```diff
@@ -0,0 +1,206 @@
+---
+title: container-log-throttling
+authors:
+  - "@syedriko"
+reviewers:
+  - "@aconway"
+  - "@portante"
+  - "@jcantrill"
+  - "@bparees"
+  - "@redhatdan"
+approvers:
+  - "@jcantrill"
+  - "@bparees"
+  - "@redhatdan"
+creation-date: 2020-02-23
+last-updated: 2020-02-23
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+see-also:
+  - conmon PR [WIP: logging behavior policies and rated-limited logging](https://github.com/containers/conmon/pull/92)
+  - libpod PR [WIP: Added support for log policy and log rate limit in conmon](https://github.com/containers/libpod/pull/4663)
+replaces: []
+superseded-by: []
+---
+
+# Container Log Throttling
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+  Container log throttling is a set of capabilities in OpenShift aimed at providing cluster administrators with means of controlling logging policies and logging throughput of individual containers or their groupings. Containers are considered sources of two log streams, stdout and stderr, that are together routed by the container runtime to a log file on the cluster node. Stdout and stderr log streams are controlled separately.
+  The following logging policies are proposed:
+  1. Passthrough
+     Unrestricted logging. For backward compatibility, this is the default if no policy is specified. It is an error to specify a rate limit with this policy.
+  1. Ignore
+     Container stdout/stderr output is not written to the log and is discarded. It is an error to specify a rate limit with this policy.
```

Comment:
same here.. we should be able to just ignore

Reply:
Sure, we can.